flying-saucer-pdf/src/test/resources/letter-spacing-wrap.html

```
@@ -0,0 +1,13 @@
+<html lang="en">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)